internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -40,8 +40,12 @@ import (
 	"github.com/SUSE/skuba/internal/pkg/skuba/deployments"
 )
 
-// defKnowHosts is the default `known_hosts` file
-const defKnowHosts = "known_hosts"
+const (
+	// defKnowHosts is the default `known_hosts` file
+	defKnowHosts = "known_hosts"
+	defSSHUser   = "root"
```

Comment:
I'm glad to change this, this was set to `root` as default so I didn't want to change it.

If we change, maybe set it to `sles` since it's what we use everywhere apart from AWS where it's `ec2-user`

Reply:
I would either change that to `sles` or, my preference, leave no default value and just fail if the user forgets to specify that.

Reply:
I wanted to default to `sles` since we start to have quite a lot of arguments to provide but then it would not be the same UX for all the platforms, I went with the node default value option ;)